content/master/cli/command-reference.md

```diff
@@ -856,135 +851,182 @@ A Kubernetes cluster running Crossplane isn't required.
+
+```yaml {label="celXRD"}
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: myXR.crossplane.io
+spec:
+# Removed for brevity
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              x-kubernetes-validations:
+              - rule: "self.minReplicas <= self.replicas && self.replicas <= self.maxReplicas"
+                message: "replicas should be in between minReplicas and maxReplicas."
+              properties:
+                minReplicas:
+                  type: integer
+                maxReplicas:
+                  type: integer
+                replicas: 
+                  type: integer
+# Removed for brevity
 ```
-{{< /expand >}}
 
-`crossplane beta validate xrd.yaml rong-xr.yaml`
-{{< expand "CEL rule violation" >}}
-```console
+The rule in this example checks that the 
+{{<hover label="celXR" line="6">}}replicas{{</hover>}} field of an XR is between
+the {{<hover label="celXR" line="7">}}minReplicas{{</hover>}} and 
+{{<hover label="celXR" line="8">}}maxReplicas{{</hover>}} values.
+
+```yaml {label="celXR"}
+apiVersion: example.crossplane.io/v1beta1
+kind: XR
+metadata:
+  name: example
+spec:
+  replicas: 49
+  minReplicas: 1
+  maxReplicas: 30
+```
+
+Running `crossplane beta validate` with the example XRD and XR produces an
+error.
+
+```shell
+`crossplane beta validate xrd.yaml xr.yaml
 [x] CEL validation error example.crossplane.io/v1beta1, Kind=XR, example : spec: Invalid value: "object": replicas should be in between minReplicas and maxReplicas.
 Total 1 resources: 0 missing schemas, 0 success cases, 1 failure cases
 ```
-{{< /expand >}}
-
-#### Validate resource against a schema
 
-Validate a provider CRD against an XRD.
+#### Validate resources against a schema
 
-`crossplane beta validate schemas.yaml resources.yaml`
+The `crossplane beta validate` command can validate an XR and one or more 
+managed resources against a provider's schema. 
 
-{{< expand "Sample validation output" >}}
-```console
-[✓] example.crossplane.io/v1beta1, Kind=XR, example validated successfully
-Total 1 resources: 0 missing schemas, 1 success cases, 0 failure cases
-controlplane $ crossplane beta validate schemas.yaml resources.yaml
-[✓] example.crossplane.io/v1beta1, Kind=XR, example validated successfully
-[✓] iam.aws.upbound.io/v1beta1, Kind=AccessKey, sample-access-key-0 validated successfully
-[✓] iam.aws.upbound.io/v1beta1, Kind=AccessKey, sample-access-key-1 validated successfully
-[✓] iam.aws.upbound.io/v1beta1, Kind=User, test-user-0 validated successfully
-[✓] iam.aws.upbound.io/v1beta1, Kind=User, test-user-1 validated successfully
-Total 5 resources: 0 missing schemas, 5 success cases, 0 failure cases
-```
-{{< /expand >}}
+{{<hint "important" >}}
+When validating against a provider the `crossplane beta validate` command
+downloads the provider package to the `--cache-dir` directory.
```

Comment:
Worth mentioning that the `--cache-dir` is called `.crossplane` by default in the current directory and move the section below to the callout?
```text
The `crossplane beta validate` command downloads and caches the schema CRD files
in the `--cache-dir` directory. By default the Crossplane CLI uses
`.crossplane/cache` as the cache location.
```